notebooks/SMAP/2.0 Read and Plot SMAP data.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>Why the Robinson projection.  It is not equal area and is not often used to display geophysical data.  It is more of a general purpose projection. </p><p> </p><p>For soil moisture, an equal area projection is probably the better projection to use. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>